it/test/controllers/agent/ChargeSummaryControllerISpec.scala

```diff
@@ -490,7 +490,8 @@ class ChargeSummaryControllerISpec extends ComponentSpecBase with FeatureSwitchi
           "documentDescription" -> "ITSA- POA 1",
           "totalAmount" -> 123456789012345.67,
           "reversalDate" -> "2020-02-24",
-          "reversalReason" -> "amended return"
+          "reversalReason" -> "amended return",
+          "poaAdjustmentReason" -> "002"
```

Comment:
if we can avoid "magic numbers" where possible =>
` "poaAdjustmentReason" -> OtherIncomeLower.code`